src/mame/atari/atarittl.cpp

```diff
@@ -293,26 +284,27 @@ void gtrak10_state::gtrak10(machine_config &config)
 	   Pixel Clock = 14.318MHz
 
 	   Horiz Total       = 451
-	   Horiz Front Porch =  0
-	   Horiz Sync        =  1
-	   Horiz Back Porch  = 31
+	   Horiz Front Porch =  ?
+	   Horiz Sync        =  32
+	   Horiz Back Porch  = ?
 
 	   Vert Total       = 521
-	   Vert Front Porch =   0
-	   Vert Sync        =   8
-	   Vert Back Porch  =   0
+	   Vert Front Porch =   ?
+	   Vert Sync        =   4
+	   Vert Back Porch  =   ?
```

Comment:
Are you sure about this?  Most monitors require exactly three lines of vertical sync and three lines of vertical front porch.  At any rate, zero for front porch and back porch is impossible.

Reply:
Yes, the COMP_SYNC signal shows 4 lines of vertical sync for field 1. For the second field it is 0.5 + 3 + 0.5 = 4 as well starting in the middle of the screen. There is no vertical front porch generated by the logic, vreset directly triggers sync.
The porches are only used to define the visible area. The physical monitor characteristics are unknown, that's why I used `?`.

Reply:
The front porch is also required for equalising pulses in interlaced modes when using composite sync. It isn't purely to define the visible area.

Reply:
The GT10 manual says that it uses a Motorola XM700 and gives a schematic.  The Lemans manual has a schematic from an XM701.  I was able to find an XM701 service manual at arcade museum by googling "xm701 service manual".

Reply:
The video parameters uses porches:
```
	m_video->set_horz_params(GTRAK10_HTOTAL  - 96, GTRAK10_HTOTAL - 64, GTRAK10_HTOTAL - 32, GTRAK10_HTOTAL);
	m_video->set_vert_params( GTRAK10_VTOTAL - 32, GTRAK10_VTOTAL -  8, GTRAK10_VTOTAL - 4,  GTRAK10_VTOTAL);
```
@goldnchild : That's great news. I am sure with the monitor manual you will get the exact porch values sorted.